pkgs/development/tools/analysis/flow/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "flow";
-  version = "0.110.0";
+  version = "0.111.0";
 
   src = fetchFromGitHub {
     owner  = "facebook";
     repo   = "flow";
     rev    = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev    = "v${version}";
```

Reply:
upstream has branches with the same name of the tags https://github.com/facebook/flow/branches, I'd prefer be explicit here to avoid accidentally resolving to a branch.